src/main/java/org/repodriller/RepositoryMining.java

```diff
@@ -90,9 +89,11 @@ public RepositoryMining process(CommitVisitor visitor) {
 	public void mine() {
 		
 		for(SCMRepository repo : repos) {
```

Comment:
This implementation might be problematic with the `threads` feature. I think we should *remove* completely the threads support. It's easy to bug and, in practice, if you want to analyze more projects, you instantiate more processes.